lib/internal/test_runner/runner.js

```diff
@@ -416,7 +416,22 @@ function watchFiles(testFiles, opts) {
   const watcher = new FilesWatcher({ __proto__: null, debounce: 200, mode: 'filter', signal: opts.signal });
   const filesWatcher = { __proto__: null, watcher, runningProcesses, runningSubtests };
   opts.root.harness.watching = true;
+  // Watch for created/deleted files in the current directory
+  const allFilesWatcher = new FilesWatcher({ __proto__: null, debounce: 200, mode: 'all', signal: opts.signal });
+  allFilesWatcher.watchPath(process.cwd());
+  allFilesWatcher.on('changed', ({ owners, eventType }) => {
```

Comment:
I think this can be added to the `watcher` object instead.

Reply:
donedone